test/clj/rems/migrations/test_convert_to_dynamic_applications.clj

```diff
@@ -3,13 +3,12 @@
             [clojure.pprint :refer [pprint]]
```

Comment:
Also don't get how this file is relevant to field changes?

Reply:
The test expectations must be changed when the label field was added to the example form.